source/beamline/CLS/CLSShutters.cpp

```diff
@@ -1,11 +1,16 @@
 #include "CLSShutters.h"
 
 #include "beamline/AMControl.h"
+#include "actions3/AMActionSupport.h"
 #include "actions3/AMListAction3.h"
 
+#include "util/AMErrorMonitor.h"
+
 CLSShutters::CLSShutters(const QString &name, QObject *parent) :
 	CLSTriStateGroup(name, parent)
 {
+	safetyShutter_ = 0;
```

Comment:
To put this in context, is this meant to be the shutter that is not controlled by beamlines or the one that is?